src/renderer/components/+workloads-overview/overview-statuses.tsx

```diff
@@ -47,7 +47,7 @@ const resources: KubeResource[] = [
 export class OverviewStatuses extends React.Component {
   @boundMethod
   renderWorkload(resource: KubeResource): React.ReactElement {
-    const store = workloadStores[resource];
+    const store = workloadStores.get(resource);
     const items = store.getAllByNs(namespaceStore.contextNamespaces);
```

Comment:
store could be `undefined` so?
```suggestion
    const items = store?.getAllByNs(namespaceStore.contextNamespaces);
```